main.tf

```diff
@@ -30,7 +30,7 @@ data "aws_iam_policy_document" "cloudtrail_assume_role" {
 
 # This role is used by CloudTrail to send logs to CloudWatch.
 resource "aws_iam_role" "cloudtrail_cloudwatch_role" {
-  name               = "cloudtrail-cloudwatch-logs-role"
+  name               = "${var.trail_name}-cloudwatch-logs-role"
```

Comment:
Do we want this? If I used this in my current deployment, this role would be "CloudTrail-xxxxxxxx-xxxx-xxxx-xxxx-2c3540565c09-cloudwatch-logs-role" which...uh...seems bad. Leaving this as "cloudtrail" seems fine to me; I'm not likely to have two cloudtrails in the same account using this module.

Reply:
mm yeah, and that will hit the 64 character limit on IAM role names. It was mostly needed to be able to run parallel tests. The two options are to use something other than trail_name or just have the tests be single threaded for now. I'm leaning towards the latter for now.